src/components/PaymentCard/PaymentCard.js

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 import { CREDIT_CARD_VISA, CREDIT_CARD_MASTER_CARD, CREDIT_CARD_AM_EX } from '../../constants/creditCardTypes';
-import creditCardType from '../../utils/creditCardType';
+import { creditCardType } from '../../utils/index';
```

Comment:
You don't have to add **index**.  `import { creditCardType } from '../../utils';` is enough.